src/org/mirah/jvm/compiler/method_state.mirah

```diff
@@ -52,9 +54,11 @@ class MethodState
       other.conflictsWith(self)
     elsif @signature.nil? || other.signature.nil?
```

Comment:
You could change this to use the new helper `isMacro`. I think that'd make it clearer.